_docs/_latest/develop/dev-setup.md

```diff
@@ -166,10 +128,10 @@ following (use the values from your terminal, not "`/SOME/PATH/TO`"):
 
 Before you run Forseti, you need to edit the forseti_conf.yaml file, found in
 `forseti-security/configs/forseti_conf.yaml`. Refer to 
-["Configuring Forseti"]({% link _docs/latest/howto/configure/configuring-forseti.md %}) 
+[Configuring Forseti]({% link _docs/latest/configure/configuring-forseti.md %}) 
 for more information.
 
-### Executing Forseti commands
+### Starting Forseti commands
```

Comment:
The word commands here with starting doesn't make sense. `Starting Forseti` is enough.

Reply:
done